code/modules/food_and_drinks/food/snacks/meat.dm

```diff
@@ -318,6 +327,10 @@
 	name = "gondola steak"
 	tastes = list("meat" = 1, "tranquility" = 1)
 
+/obj/item/reagent_containers/food/snacks/meat/steak/penguin
+	name = "penguin steak"
+	tastes = list("a weird mix of beef and cod fish" = 1)
```

Comment:
Please use a taste value more like the other tastes, you could just make it have
```suggestion
	tastes = list("beef" = 1, "cod fish" = 1)
```